packages/schemas/src/utils.ts

```diff
@@ -103,6 +103,50 @@ export const hex2RgbColor = (hexString: string | undefined) => {
   return undefined;
 };
 
+export const hex2CmykColor = (hexString: string | undefined) => {
+  if (hexString) {
+    const isValid = isHexValid(hexString);
+
+    if (!isValid) {
+      throw new Error(`Invalid hex color value ${hexString}`);
+    }
+
+    // Remove the # if it's present
+    hexString = hexString.replace('#', '');
+
+    // Extract the hexadecimal color code and the opacity
+    var hexColor = hexString.substring(0, 6);
```

Comment:
Please use `const` or `let` instead of `var`

Reply:
Thanks. I have updated.